### DISCUSSION TITLE
Contribution question.

### DISCUSSION THREAD

**Discussion by Burekslayer** [PankajSingh2003/NASA_startingpage#36] [Q&A] [answered]
Hello, I've seen that you have 2 issues with responsiveness (I'm assigned to one of them), so I'm wondering if I should still try to do it, or should I do the other issue I got assigned to? (footer issue)

**Comment by PankajSingh2003** [comment, marked as answer]
you can do the footer issue if you want to.
Your efforts are appreciated in advanced.

**Reply by PankajSingh2003** [reply]
Well you can make footer like these or can try your way of making footer if you want.
![footer](https://user-images.githubusercontent.com/104151308/196675312-e9a41083-1089-41d2-8c76-9e8b8ac099fe.png)

**Reply by Burekslayer** [reply]
Will do, and for the links, are there any of the links for these social media icons or I just need to make a design?
